src/contents/all/components/app/App.tsx

```diff
@@ -199,78 +204,110 @@ const App = () => {
          ({document, window}) => {
             // Render Children
             return (
-              <div className={`analogueModal ${content ? "loaded" : ""}`} onClick={(e) => {
-                e.stopPropagation()
-              }}>
-
-                <div className="analogueModalHeader" id='analogueHeader'>
-
-                  <img src={logo} className="logo" alt="Analogue Icon" />
-
-                  <Dropdown
-                    disabled={!log}
-                    align={{offset: [-14, 15]}}
-                    overlayClassName="dropdownStatusOverlay"
-                    getPopupContainer={() => document.getElementById("analogueHeader")}
-                    overlay={
-                      <Menu onClick={updateLogStatus}>
-                        {log && log.status !== "pub" &&
-                          <Menu.Item key="pub">
-                            Add to library
-                          </Menu.Item>
-                        }
-                        {log && log.status !== "saved" &&
-                          <Menu.Item key="saved">
-                            Save for later
-                          </Menu.Item>
-                        }
-                        {log && log.status !== "priv" &&
-                          <Menu.Item key="priv">
-                            Add privately
-                          </Menu.Item>
-                        }
+              <>
+                <div className={`analogueModal ${content ? "loaded" : ""}`} onClick={(e) => {
+                  e.stopPropagation()
+                }}>
+                  {login
+                    ? (
+                      <>
+                        <div className="analogueLoginHeader">
+
+                        <a
+                          target="_blank"
+                          href={`${process.env.NODE_ENV === 'production' ? 'https://www.analogue.app' : 'http://localhost:3000'}`}
+                        >
+                          <img src={logo_long} className="logo" alt="Analogue Logo"/>
```

Comment:
Can't seem to render the logo in the extension. For some reason, it tried to load "logo" from the background page (divisare.com). Probably an easy fix but couldn't figure it out for the life of me.